### PR TITLE
net: ethernet: add PTP multicast default MAC address

### PR DESCRIPTION
**Pull request by Cherish-Gww**
All except peer delay message in PTP protocol use multicast MAC
address is 01:1b:19:00:00:00

### CODE REVIEW

**Review by rlubos — COMMENTED** [review]
@Cherish-Gww Please fix the compliance issues:
* there should be no `subsys:` in the commit title prefix
* the commit body line is too long

**Comment by Cherish-Gww** [comment]
> @Cherish-Gww Please fix the compliance issues:
> 
> * there should be no `subsys:` in the commit title prefix
> * the commit body line is too long

Done

**Review by rlubos — APPROVED** [review]

File: include/zephyr/net/ethernet.h [hunk 1]
```diff
@@ -715,6 +715,22 @@ static inline bool net_eth_is_addr_lldp_multicast(struct net_eth_addr *addr)
 	return false;
```
Comment (jukkar) [inline include/zephyr/net/ethernet.h:715]: Typo in the commit msg s/except/expect/

Comment (jukkar) [reply include/zephyr/net/ethernet.h:715]: Also the commit subject should start with `net: ethernet: ...` as this is Ethernet specific change and not really related to generic l2 layer.

Comment (Cherish-Gww) [reply include/zephyr/net/ethernet.h:715]: > Typo in the commit msg s/except/expect/
Thank you @jukkar I have changed the relevant commit message

In wiki: https://en.wikipedia.org/wiki/Precision_Time_Protocol
Multicast group address table:  All except peer delay messages

Comment (jukkar) [reply include/zephyr/net/ethernet.h:715]: Ok, my bad, I thought it was a typo.

**Review by jukkar — APPROVED** [review]

**Pull request merged**